src/Service/EntityService/Product/ProductService.php

```diff
@@ -6,14 +6,15 @@
 use App\Service\EntityService\BaseImportInterface;
 use App\Traits\EntityManagerTrait;
 use DateTime;
+use InvalidArgumentException;
 use Symfony\Component\Validator\ConstraintViolationListInterface;
 use Symfony\Component\Validator\Validator\ValidatorInterface;
 use Symfony\Component\Validator\Constraints as Assert;
 
 class ProductService implements BaseImportInterface {
     use EntityManagerTrait;
 
-    private const PRODUCT_HEADERS = [self::PRODUCT_NAME, self::PRODUCT_CODE, self::PRODUCT_DESCRIPTION, self::STOCK, self::COST_IN_GBP, self::DISCONTINUED];
+    public const PRODUCT_HEADERS = [self::PRODUCT_NAME, self::PRODUCT_CODE, self::PRODUCT_DESCRIPTION, self::STOCK, self::COST_IN_GBP, self::DISCONTINUED];
```

Comment:
I don't think it has to be public. You can override access in the mock for testing if needs